src/wormhole_transit_relay/transit_server.py

```diff
@@ -46,7 +46,10 @@ def describeToken(self):
     def connectionMade(self):
         self._started = time.time()
         self._log_requests = self.factory._log_requests
-        self.transport.setTcpKeepAlive(True)
+        try:
+            self.transport.setTcpKeepAlive(True)
```

Comment:
The string transport used during testing doesn't have this method (because it's not a TCP transport)